cmd/aio/main.go

```diff
@@ -83,11 +84,18 @@ func main() {
 			hcloudClient,
 		),
 	)
+
+	extraVolumeLabels, err := utils.ConvertLabelsToMap(os.Getenv("HCLOUD_VOLUME_EXTRA_LABELS"))
+	if err != nil {
+		logger.Error("could not parse extra labels for volumes")
+		os.Exit(1)
+	}
```

Comment:
Please make sure to log the error that happened during parsing. Otherwise, the user will not know how to fix the issue.

(Also applies to `cmd/controller/main.go`)